uLaunch/source/cfg/cfg_Config.cpp

```diff
@@ -384,6 +384,13 @@ namespace cfg {
         ofs.close();
     }
 
+    void DeleteRecord(const TitleRecord record){
```

Comment:
`) {` in the function

(sorry if I get too annoying with this details, otherwise I can merge this and change them myself)

Reply:
no problems, i will change it now according to your style :)